mote/projects/base/library/browser/patterns/atoms/picture/data.yaml

```diff
@@ -1,21 +1,23 @@
-attrs:
-    data-foo: ''
-sources:
-    - srcsets:
-        -
-            src: foo@200.webp
-            dimensions: 200w 100h
-        -
-            src: foo@400.webp
-            dimensions: 400w 200h
-    - srcsets:
-        -
-            src: foo@200.png
-            dimensions: 200w 100h
-        -
-            src: foo@400.png
-            dimensions: 400w 200h
+#sources:
```

Comment:
I commented out all of this to ensure that dummy data does not leak through when used in production.